services/src/main/java/uk/gov/moj/sdt/services/BulkSubmissionService.java

```diff
@@ -31,14 +31,20 @@
 package uk.gov.moj.sdt.services;
 
 import java.util.ArrayList;
+import java.util.HashMap;
```

Comment:
This import is not used.  Remove it if it is not needed.

Reply:
removed